### DISCUSSION TITLE
Welcome to QuestJS Discussions!

### DISCUSSION THREAD

**Discussion by ThePix** [ThePix/QuestJS#108] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Woodhouse3D** [comment]
Hello, I'm Woodhouse3D.
I've been working on a QuestJS project with complex RPG-like narrative mechanics.
I'm working on it to learn and practice Web development skills, and I'm very interested in the possibility of contributing to the code base.

**Comment by KVonGit** [comment]
Hi, I'm KV.

I create and solve problems.

---
I also post as TextMisadventures from time to time, as I use that account for Quest 5 testing purposes, and I sometimes forget to check which account I'm under before posting things. `:)`

**Comment by Pertex** [comment]
Hi ThePix,
good to see you back again!

**Comment by kyrathasoft** [comment]
I write fiction. Current WIP is a choice based story-game (with concurrently written tutorial) using Twine and Sugarcube. It’s a campfire tale turned otherworldly survival fiction. Very interested in QuestJS. Is there a manual, or are there user-authored tutorials?

**Comment by ThePix** [comment]
Hi kyrathasoft

The documentation is [here](https://github.com/ThePix/QuestJS/wiki), and there is a tutorial [here](https://github.com/ThePix/QuestJS/wiki/Tutorial-1-%E2%80%90-First-steps).

**Comment by GhostOfMoria** [comment]
Hi there! Been quite a while. I was building a big 'ol game using lots of your tutorials on this page: https://github.com/ThePix/quest/wiki I had many things working together that I was quite proud of, including a working torch system that could illuminate rooms and lasted a certain amount of time. I integrated the CombatLib, time system, Quest tracking, shops and stackable items, and created NPC's with autonomous movement and actions. However, I abandoned my project when I realized what a headache it would be to allow others to play it due to Quest's compatibility and accessibility restrictions. But making this all run in JS eliminates that issue, right? 

I'm starting to work my way through your tutorials now, and I'm excited to reignite my project, even though it means basically starting from scratch. I just wanted to drop you a line and say thank you for keeping this project alive and making it possible to continue development. And thank you for being so generous with your tutorials and knowledge. Truly excited to get back into it.
